src/Datadog.Trace/ExtensionMethods/RandomExtensions.cs

```diff
@@ -4,6 +4,42 @@ namespace Datadog.Trace.ExtensionMethods
 {
     internal static class RandomExtensions
     {
+#if !NETSTANDARD
+        private static readonly Random GlobalRandom = new Random();
+#endif
+
+        [ThreadStatic]
+        private static Random _threadRandom;
+
+        public static Random GetRandom()
+        {
+            var random = _threadRandom;
```

Comment:
lease do not use var!!

Reply:
Did we reach an agreement on that? Right now all our shared configs for R# and editorconfig will suggest using `var` in this case.

Reply:
I don't believe we reached an agreement

Reply:
Let's discuss this. I am very passionate about not using "var" in a strongly typed language, but it the team really wants it, I do not want to force. Either way, let's me consistent.

Reply:
That's precisely because this is a strongly typed language that I believe var should be used :) The same way we don't have to manually track object lifetime in a managed language, we don't have to burden ourselves with type information in a strongly typed language, because the compiler will take care of it and guarantee us that our code is safe.